src/infi/__init__.py

```diff
@@ -1 +1,3 @@
-__import__("pkg_resources").declare_namespace(__name__)
+import sys
+if sys.hexversion < 0x30300f0:
```

Comment:
Why not use `sys.version` here? https://stackoverflow.com/questions/1093322/

My main concern here is that this doesn't look very good ;)